playbooks/eco.yml

```diff
@@ -5,7 +5,7 @@
   vars:
     cache_dir: "{{ (lookup('env','HOME'), '.cache', 'ansible-lint-eco') | path_join | realpath }}"
     repos:
-      - name: ansible-role-mysql
+      - name: geerlingguy.mysql
```

Comment:
Testing that we can lint role folders that use the galaxy naming when installed is a good idea but removing the old test  about `ansible-role-mysql` format is also a very bad idea, as we could easily break it. As this role is small, I do not see any problem in cloning it twice with both names, so we assure both cases are covered.

Reply:
the problem is that keeping the current configuration will lead to CI failure as the ``.ansible-lint`` file of this role is configured with ``role-name`` in  ``skip_list``.